builder/bundleset_test.go

```diff
@@ -81,6 +99,28 @@ pkg1 # [TITLE]: wrongtitle
 			},
 			ExpectedIncludes: []string{"a"},
 			ExpectedPackages: map[string]bool{"pkg1": true},
+			ExpectedChroots:  map[string]bool{},
+		},
+		{
+			Contents: []byte(fmt.Sprintf(`# Duplicate content chroots and trailing /s
+# [TITLE]: fake
+# [DESCRIPTION]: a description
+# [STATUS]: a status
+# [CAPABILITIES]: the capabilities
+# [MAINTAINER]: the maintainer
+content(%s/)
+content(%s)
+content(%s///)
```

Comment:
@reaganlo I added this unit test to verify that content chroots can handle trailing slashes.